scripts/app.js

```diff
@@ -720,6 +730,7 @@
 				this.payPeriodView = false;
 				this.metricsView = false;
 				this.starredCasesView = false;
+				this.settingsView = false;
```

Comment:
dupe

Reply:
how is this one a dupe?

Reply:
On line 733 `this.settingsView = false`.
Then on line 745 `this.settingsView = false;`